### ISSUE TITLE
Support Browser Mod 2.0

### ISSUE THREAD

**Issue by conorlap** [DBuit/Homekit-panel-card#142]
Browser Mod has released a breaking change release which means all things that utilize browser_mod need updated.

Further info:
https://github.com/thomasloven/hass-browser_mod

**Comment by jimmyhawkin** [comment]
Second that.

**Comment by tenallero** [comment]
Me too

**Comment by tenallero** [comment]
I have installed the latest release, including browser-mod, .... and it does not work. The popup does not appears

**Comment by DBuit** [comment]
@tenallero can you share your configuration?
The updated browser mod is working good? You have browser_mod visible in your sidebar?

**Comment by conorlap** [comment]
@tenallero have you cleared your browser cache?

**Comment by tenallero** [comment]
Yes, I have cleared browser cache. Chrome, Firefox and iPhone HomeAssistant app.
Now I am trying to remove and re-install both components

**Comment by conorlap** [comment]
@tenallero I had to delete Browser Mod from my integrations, restart my Home Assistant and then add Browser Mod under integrations again - this made sure it was the new Browser Mod.

**Comment by tenallero** [comment]
@conorlap 
@DBuit 

After removing and reinstalling ... Now, I can say popup is working with browser_mod 2.0
Thanks to all

browser_mod 2.1.3
homekit_panel_card 0.6.3

Note: homekit_panel_card still is showing 0.6.2 at browser´s console output

**Comment by cpsskipper** [comment]
After removing and reinstalling ... missing transparency on popup ((
